lib/rules/unit-no-unknown/index.js

```diff
@@ -116,8 +117,10 @@ function rule(actual, options) {
 							return;
 						}
 
-						if (/^image-set/i.test(value)) {
-							const imageSet = parsedValue.nodes.find((node) => node.value === 'image-set');
+						if (/^(?:-webkit-)?image-set[\s(]/i.test(value)) {
```

Comment:
[IMO] If using [`\b` (word boundary)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Assertions#Types), the intent of the code may be clear.

For example:

```suggestion
						if (/^(?:-webkit-)?image-set\b/i.test(value)) {
```